hack/run-e2e-kind.sh

```diff
@@ -4,9 +4,11 @@ export ROOT_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/..
 export LOG_LEVEL=3
 export CLEANUP_CLUSTER=${CLEANUP_CLUSTER:-1}
 export CLUSTER_CONTEXT="--name test"
-export IMAGE="nginx:latest"
+export IMAGE_NGINX="nginx:latest"
+export IMAGE_BUSYBOX="busybox:latest"
 export KIND_OPT=${KIND_OPT:=" --config ${ROOT_DIR}/hack/e2e-kind-config.yaml"}
 export KA_BIN=_output/bin
+export WAIT_TIME="--wait 20s"
```

Comment:
only `20s` there, move `--wait` to `kind` command.

Reply:
Raised New PR for that change